src/panels/lovelace/editor/config-elements/hui-logbook-card-editor.ts

```diff
@@ -80,6 +84,18 @@ export class HuiLogbookCardEditor
     );
   }
 
+  private async _loadNumericDeviceClasses(hass: HomeAssistant) {
+    this._sensorNumericDeviceClasses = [];
```

Comment:
What is the purpose of this? In case `getSensorNumericDeviceClasses` fails?

Reply:
This ensures that the _load function is not called a second time if another `updated` occurs before the async function returns. 

The `[]` is truthy so the load function will not be called a second time while it is waiting for the await to complete.